packages/main/test/specs/ComboBox.spec.js

```diff
@@ -69,7 +69,11 @@ describe("General interaction", () => {
 		await input.click();
 		await input.keys("b");
 
-		await browser.pause(200);
+		await browser.waitUntil(() => popover.getProperty("opened"), {
+			timeout: 200,
+			timeoutMsg: "Popover should be displayed"
+		});
+
 		assert.ok(await popover.getProperty("opened"), "Popover should be displayed");
```

Comment:
Isn't that assert now redundant and covered with the change made?

Reply:
Yes, thanks for  pointing this out. I'll fix it in a separate change